step-templates/aws-rds-sql-backup-s3.json

```diff
@@ -0,0 +1,83 @@
+  },
+  "Parameters": [
+    {
+      "Id": "3e45bb88-3632-4115-a0d5-54680615f0ca",
+      "Name": "rdsSqlBackupbackupSqlServer",
+      "Label": "SQL Server",
+      "HelpText": "The SQL Server to perform the work on",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "880a4e73-64fe-4a36-b4c3-a281b64e3c23",
+      "Name": "rdsSqlBackupbackupSqlLoginUserWhoHasCreateUserRights",
+      "Label": "SQL Login",
+      "HelpText": "The login of the user who has permissions to create a database.\n\nLeave blank for integrated security",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "6689fbe5-f47a-4800-945d-df50fc19c7b0",
+      "Name": "rdsSqlBackupbackupSqlLoginPasswordWhoHasRights",
+      "Label": "SQL Password",
+      "HelpText": "The password of the user who has permissions to create SQL Logins\n\nLeave blank for integrated security",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "Sensitive"
+      }
+    },
+    {
+      "Id": "1a358cca-5cd1-41ae-b763-fcaf2c7350f9",
+      "Name": "rdsSqlBackupbackupDatabaseName",
+      "Label": "Database Name",
+      "HelpText": "The name of the database to create the user on",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "c5e2fa69-1b42-4427-91d2-10e1a13af744",
+      "Name": "rdsSqlBackupbackupS3Bucket",
+      "Label": "S3 Bucket Name",
+      "HelpText": "The name of the bucket (including any sub directories).",
+      "DefaultValue": "",
+      "DisplaySettings": {
+        "Octopus.ControlType": "SingleLineText"
+      }
+    },
+    {
+      "Id": "ea9d9eb8-9c0f-4c69-a6d4-4d00e43383af",
+      "Name": "rdsSqlBackuprdsSqlBackup",
```

Comment:
This parameter is not used in the script. I think its a typo? and should be `rdsSqlBackupbackupFileName`